cookbook/integrations/aws/athena/athena.py

```diff
@@ -98,7 +98,7 @@ def no_io_wf():
 @task
 def manipulate_athena_schema(s: FlyteSchema) -> FlyteSchema:
     df = s.open().all()
-    return df[df.total_vaccinations.notnull()]
+    return df
```

Comment:
I think we also need to adjust SQL query to 
```sql
SELECT * FROM vaccinations where iso_code like '{{ .Inputs.country_code }}'
```

Reply:
update it, thanks